### DISCUSSION TITLE
External URI link and dialog box

### DISCUSSION THREAD

**Discussion by GeorgesCaen** [bluesky-social/atproto#2951] [Clarifications and "Why?"]
Hello. I understand that the dialog box for confirmation appears for security reasons.
If I create a facet with such an external https URL link, it works and the dialog box appears when clicking.
But if I type an the same URL directly in a post, there is no dialog box.

Is there a way to prevent this dialog box to appear  without creating other problems ?

**Comment by searls** [comment]
Same question -- I imagine this is because the domain is self-evident when looking at a link title as URL that matches the destination, but a link facet with a custom text label could mislead someone to believe they're going to another domain. It's a real problem, but one that I wish could be handled more gracefully.
